plugins/rabbitmq_logs.yaml

```diff
@@ -0,0 +1,34 @@
+version: 0.0.1
+title: RabbitMQ
+description: Log parser for RabbitMQ
+parameters:
+  - name: daemon_log_path
+    type: string
+    default: "/var/log/rabbitmq/rabbit@*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    filelog:
+      include: [ '{{ .daemon_log_path }}' ]
+      multiline:
+        line_start_pattern: '\d+-\d+-\d+ \d+:\d+:\d+\.\d+\+\d+:\d+'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: rabbitmq
+      operators:
+        - type: regex_parser
+          regex: '^(?P<timestamp>\d+-\d+-\d+ \d+:\d+:\d+\.\d+\+\d+:\d+) \[(?P<severity>\w+)\] \<(?P<process_id>\d+\.\d+\.\d+)\> (?P<message>.*)'
```

Comment:
This regex for the timestamp is fairly weak (too permissive). I can provide a better regex that limits to the correct max lengths for fields so that this cannot trigger accidentally. I just need to see a sample or three. :)

Essentially, if the time for example is always 2 digits (02:07:55 for example), then the time piece should be \d{2}:\d{2}:... This prevents false triggering, a serious danger with using the `+` in regex too freely.

Reply:
updated this. Left the millisecond as `\d+` as I was confident that precision was always going to be that many digits long.